pyseir/inference/model_fitter.py

```diff
@@ -486,7 +543,11 @@ def _fit_seir(self, R0, t0, eps, t_break, test_fraction, hosp_fraction, log10_I_
         self.dof_deaths = (self.observed_new_deaths > 0).sum()
         self.dof_cases = (self.observed_new_cases > 0).sum()
 
-        return chi2_deaths + chi2_cases + chi2_hosp
+        not_penalized_score = chi2_deaths + chi2_cases + chi2_hosp
+        # Calculate the final score as the product of the future_days_penalty and not_penalized_score
+        score = future_days_penalty * (chi2_deaths + chi2_cases + chi2_hosp)
```

Comment:
Seems this should be additive not multiplicative. else you are putting a very strong and non-convex prior on not going over the fit time. This might be too aggressive to fit recent data, and doesn't have easy to understand tradeoffs against the other chi2.

Reply:
PR #2